notebooks/tests/test_nessie_iceberg_hive_demo_nba.py

```diff
@@ -29,7 +29,8 @@
 from . import start_nessie
 
 num_salaries_on_experiment = "59"
-num_salaries_on_main = "55"
+# TODO: change this back to 55 when we remove the workaround for https://github.com/apache/iceberg/pull/4509
+num_salaries_on_main = "56"
```

Comment:
as commented, this changes because of the workaround where we insert a single row on the `main` branch, to make sure we can count the rows afterwards without error.